### PR TITLE
Add AI Training control panel

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5457, MANAGER-5831
| License          | BSD 3-Clause

### :rocket: Features

Add AI Training control panel

### :link: Relates

- Initial Pull Request: #3459

### :flags: Translations

- [x] TRANS-28537
- [x] TRANS-28574
- [x] TRANS-28753
- [x] TRANS-30135
- [x] TRANS-30312

### :handshake: Contributors

@XciD @mael-le-gal @co42 @christophe-rannou

### CODE REVIEW

**Comment by antleblanc** [comment]
Rewriting git history.

**Comment by antleblanc** [comment]
Rebasing on `develop` in order to rewrite the commit history.

**Comment by antleblanc** [comment]
Rebased successfully on `develop`.

File: packages/manager/modules/pci/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ovh-ux/manager-pci",
-  "version": "5.1.4",
+  "version": "5.1.2",
```
Comment (antleblanc) [inline packages/manager/modules/pci/package.json:3]: **note:** Must be fixup.

**Comment by marie-j** [comment]
@antleblanc I think given the commit history you might need to `--autosquash` while rebasing

**Comment by antleblanc** [comment]
> @antleblanc I think given the commit history you might need to `--autosquash` while rebasing

Yes! You're right! I'm currently validating changes with @christophe-rannou!
Once it will be fixed, then I will rebase and also be able to send a translation request based on recent changes.

Thank you!

**Comment by antleblanc** [comment]
Rebasing on `master` branch.

**Comment by antleblanc** [comment]
Rebased successfully on `master` branch.

**Pull request merged**